bin/maestro.js

```diff
@@ -18,6 +19,10 @@ switch(argv._[0]) {
   case 'teardown':
     teardown(argv);
     break;
+  // TODO: perhaps `maestro get-templates` should be called as part of the `maestro config` command? If so, should we still leave `get-templates` as a top level sub-command?
+  case 'get-templates':
```

Comment:
I think that if we've separated the templates to their own repository, it makes sense to have a separate `get-templates` command. I can see the logic to go the other way and integrate this into `git config` though.